package.json

```diff
@@ -22,6 +24,7 @@
     "test": "watai test && watai test/bootstrap-compat",
     "precompile": "rimraf dist/*",
     "compile": "webpack --optimize-minimize",
-    "compile-continuously": "webpack --watch"
+    "compile-continuously": "webpack --watch",
```

Comment:
Ok, then remove `compile-continuously`, as compiling in itself is a distribution act.